cha/introduction/sota_app.tex

```diff
@@ -0,0 +1,254 @@
+When that music is added, Chorus streams the music to connected slave devices.
+To connect to a master device playing music, you press the menu button in the right top corner, and press join.
+Here a menu pops up with devices playing on the network, and if you select the device shown, it connects as a slave and starts playing.
+We encountered a scenario where the app just kept loading when attempting to join an active network, after a restart of the app it worked.
+
+It is intuitive to use the app, it is very minimalistic hence easy to navigate.
+The design is a bit old but its simplicity makes up for it.
+
+Chorus supports manual and automatic synchronization.
+On \cref{fig:chorus_screenshot} the synchronization slider can be seen,
+it slides from $-400ms$ to $+400ms$, and each step is $\Delta 10ms$.
+
+The app is free to download and use without any adds.\cite{chrous_play} 
+
+\begin{figure}[h!]
+    \setlength{\fboxsep}{0pt}
+    \setlength{\fboxrule}{2pt}
+    \begin{minipage}[b]{0.45\textwidth}
+        \centering
+        \fbox{\includegraphics[width=0.65\textwidth]{img/sota/chorus.png}}
+        \caption{A screenshot of Chorus when the app is opened.}\label{fig:chorus_screenshot}
+    \end{minipage}
+    \hfill
+    \begin{minipage}[b]{0.45\textwidth}
+        \centering
+        \fbox{\includegraphics[width=0.65\textwidth]{img/sota/chorus_slider.png}}
+        \caption{A screenshot of Chorus' synchronization slider.}\label{fig:chorus_slider}
+    \end{minipage}
+\end{figure}
+
+\iffalse
+By AVR APPS 
+3.3 stars on Google Play
+Functionality / How good it works
+11. maj 2015
+Android 4.0, iOS
+Local media?
+Mobile hotspot or WiFi
+Free
+Manual/automatic sync
+\fi
+\subsection{Determining the State of the Art}
+To determine the state of the art in regard to Android apps, we compare the described apps.
+In \cref{tab:sota_comp}, a comparison of the apps can be seen. 
+
+In regard to SoundSeeder versus AmpMe, then in the rating, last update, and supported devices, they are pretty equal.
+When it comes to the media source, they both supports YouTube and local media. 
+AmpMe supports Spotify, which is very popular, but SoundSeeder supports more different and basic technologies,
+like DLNA and UPnP, which makes it possible to e.g. stream from a media server.
+SoundSeeder also supports the possibility of using an external device as media source,
+which could make it possible to use another device to play Spotity (or SoundCloud for that matter) to the master device,
+and then stream it to the connected slaves. 
+This could maybe impair the mobility, but that is very dependent on the situation.
+
+On the other hand, AmpMe does not require the devices to be on the same network, only to have an Internet connection,
+where SoundSeeder requires the devices to be on the same network.
+This can be an annoyance in the case where there is no WiFi available and all devices have to be on the same access point.

```

Comment:
Dette er forvirrende - hvilken en af de to er en annoyance? SoundSeeder virker da superier her eftersom du kan lave et ad-hoc network men du nævner det er en annoyance at alle skal være på samme AP, så hvad refere "this" til?